CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Fix public node resolution for connectivity indicator ([#4205]https://github.com/hoprnet/hoprnet/pull/4205)
 - Remove charset complexity validation on API token ([#4210](https://github.com/hoprnet/hoprnet/pull/4210))
 - Properly encode API token passed from the Admin UI ([#4210](https://github.com/hoprnet/hoprnet/pull/4210))
+- Refactor timeouts for more throughput and increase usage of iterables ([#4238](https://github.com/hoprnet/hoprnet/pull/4238))
```

Comment:
Only a single PR was merged since the last back-merge? Are changelogs missing?

Reply:
I think only @robertkiel 's PRs were merged, maybe some of those would deserve another changelog entry.